### DISCUSSION TITLE
Filtering of Aspect Types is "lost in translation"?

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#594] [General] [answered]
It seams that filtering or selecting Aspect Types doesn't work anymore in the Dutch modus. For example: selecting Aspect Types "Kennis" gives no output for Aspects which are created with Aspect Type "Knowledge".
Note: it even looks like Aspect Types are not translated when switching to a different language.

![afbeelding](https://user-images.githubusercontent.com/102361398/165759099-06474e24-cead-49b0-9121-5393d6742612.png)

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof, 

Yes this is something we will need to address. Aspect types themselves actually are not "translateable" at the moment as they are data driven. 

So the fix we will be offering near term is to be able to update the type on the settings - so that admins can then switch from "Knowledge" to "Kennis". If the number of aspects is low this will work, and then we will need to think about having aspect definitions support multiple languages later...

Would the above be ok?

Cheers, Neil

**Reply by GerlofdeHaan** [reply]
Hi Neil,

Updating the type on the settings sounds sufficient. 
So thanks for that.

Gr. Gerlof
